web/docs/introduction/getting-started.md

```diff
@@ -107,12 +107,11 @@ Open your terminal and run:
 curl -sSL https://get.wasp-lang.dev/installer.sh | sh
 ```
 
-If you are using macOS on a device with Apple Silicon you might encounter `Bad CPU type in executable` issue. To fix that, please [install Rosetta on your Mac](https://support.apple.com/en-us/HT211861).
-
-```shell
-softwareupdate --install-rosetta
-```
+:::note Something Unclear?
```

Comment:
I merely copied the note block from above in the file with "Something unclear?", I didn't mean this title to be kept for here, my bad I should have made that clear!
Here is suggested change from my side:

```suggestion
:::note Running Wasp on Mac with Mx chip (arm64)
```